MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -1136,20 +1136,20 @@ private void FetchFromRatingsNode(XmlReader reader, T item)
                     switch (reader.Name)
                     {
                         case "rating":
-                        {
-                            if (reader.IsEmptyElement)
                             {
-                                reader.Read();
-                                continue;
-                            }
+                                if (reader.IsEmptyElement)
+                                {
+                                    reader.Read();
+                                    continue;
+                                }
 
-                            var ratingName = reader.GetAttribute("name");
+                                var ratingName = reader.GetAttribute("name");
 
-                            using var subtree = reader.ReadSubtree();
-                            FetchFromRatingNode(subtree, item, ratingName);
+                                using var subtree = reader.ReadSubtree();
+                                FetchFromRatingNode(subtree, item, ratingName);
 
-                            break;
-                        }
+                                break;
+                            }
```

Comment:
Just remove the braces

Reply:
We use braces everywhere else in this class, so remove them in the other switch cases too?

Reply:
In that case you can just leave it